.eslintrc.json

```diff
@@ -3,5 +3,8 @@
   "settings": {
     "mdx/code-blocks": true,
     "mdx/language-mapper": {}
+  },
+  "rules": {
+    // define own rules
```

Comment:
?

Reply:
while running `yarn run lint-mdx --config "$MDX_LINT_CONFIG"` in this 
`--config "$MDX_LINT_CONFIG"` will check the `.eslintrc.json` file and whatever configuration is in rules it will write that.
for now idk what linting rules are used in this project for `.mdx` , so I left it empty and after review I'll make PR.

Reply:
Idk what linting rule this repo has, so I left is empty

Reply:
hey, taking a look at this and running it on my own fork, to verify it works. unless you've already done this?